frontend/src/AccountForm.js

```diff
@@ -0,0 +1,59 @@
+import React, {useState} from 'react';
+
+function AccountForm(props) {
+  const [person, setPerson] = useState(
+     {
+        username: "",
```

Comment:
We want the users to provide a Firstname and Lastname, a username will not be used